spec/spec_helper.rb

```diff
@@ -1,8 +1,43 @@
+ENV['RACK_ENV'] = 'test'
+
 require 'coveralls'
 require 'simplecov'
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+require 'data_mapper'
+# require 'web_helper'
+require 'database_cleaner'
+require File.join(File.dirname(__FILE__), '..', 'app/app.rb')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.